src/background/service/preference.ts

```diff
@@ -10,6 +10,14 @@ export interface Account {
   address: string;
   brandName: string;
 }
+export interface ChainGas {
+  gasPrice?: number; // custom cached gas price
+  gasLevel?: 'slow' | 'standard' | 'fast'; // cached gasLevel
```

Comment:
Just checked the gasMarket API, should be `slow | normal | fast`

Reply:
done